src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -259,7 +259,10 @@ private void ListView_SizeChanged(object sender, SizeChangedEventArgs e)
             }
 
             var newWidth = _projectColumnHeader.ActualWidth + width;
-            _projectColumn.Width = newWidth;
+            if (newWidth > 0)
```

Comment:
This is probably the most failsafe change to catch the problem before setting the property.
 I wonder if there is another change in the calculations above that should be done. If ActualWidth is <8, then `width` is negative upon initialization. So, all remaining logic will just make the value more negative.